test/crltest.c

```diff
@@ -239,7 +240,11 @@ static int Verify(X509 *leaf, STACK_OF(X509) *roots, STACK_OF(X509_CRL) *crls,
     X509_STORE_CTX_set0_param(ctx, param);
 
     ERR_clear_error();
-    if (X509_verify_cert(ctx) != 1) {
+    retcode = X509_verify_cert(ctx);
+    /* Unset the CA and CRL stacks so they don't get free'd by us. */
+    X509_STORE_CTX_trusted_stack(ctx, NULL);
+    X509_STORE_CTX_set0_crls(ctx, NULL);
```

Comment:
doh, of course i wasn't making sense. :(  backed it out.  need to reproduce locally.